src/components/05_pages/Content/Content.js

```diff
@@ -206,7 +205,6 @@ class Content extends Component {
                             });
                           }}
                           margin="normal"
-                          disabled={!this.props.contentList.length || false}
```

Comment:
I had specifically disabled these to prevent interaction if there was no content to be filtered. I don't think it makes sense to allow a user to filter 'nothing'.

Reply:
The problem is that you might select a set of filters that is too restrictive and doesn't find any results - and after that, the filters would be disabled meaning that only way to move back to less restrictive filter set is to refresh the page 😅

Reply:
Makes sense!

Reply:
Oh great point!